docs/changelog/89803.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 89803
+summary: Fork `TransportRollupCapsAction` to MANAGEMENT POOL
```

Comment:
Is it important to mention the exact names of the two affected Transport Actions here?